docs/api/ref/schemas/knowledge_panels/panel.yaml

```diff
@@ -13,6 +13,20 @@ properties:
   expand_for:
     type: string
     description: 'If set to "large", the content of the panel should be expanded on large screens, but it should still be possible to unexpand it.'
+  name:
```

Comment:
for name and description, I think it would fit best in the title element where we already have a title / sub_title:
https://github.com/openfoodfacts/openfoodfacts-server/blob/main/docs/api/ref/schemas/knowledge_panels/elements/title_element.yaml

Currently we have:

```
title_element: {
icon_size: "small",
title_element: {
icon_size: "small",
icon_url: "https://static.openfoodfacts.org/images/misc/high.svg",
title: "Saturated fat in high quantity (10.6%)"
},icon_url: "https://static.openfoodfacts.org/images/misc/high.svg",
title: "Saturated fat in high quantity (10.6%)"
},
```

We could add "name", "value" in there, as they would be displayed as the clickable knowledge panel title.

We could also add "value_label" or similar so that we can display the "Additives to avoid" in the mockup:

![image](https://github.com/openfoodfacts/openfoodfacts-server/assets/8158668/18fa5bce-0f10-4656-a5cf-37e5cb153ba1)

Reply:
Aren't the "Additives to avoid" just names too? e.g. we could equally well display it as:

Additives to avoid >
 (red circle) 0

Reply:
Indeed, in terms of data, both the Fat and Additive to avoid panels are the same. There's a "field" name (fat or additive to avoid) and a value, possibly a unit (%), and an evaluation. So no need for "value_label"